plugin.audio.soundcloud/resources/plugin.py

```diff
@@ -0,0 +1,184 @@
+from resources.lib.soundcloud.api_v2 import ApiV2
+from resources.lib.kodi.cache import Cache
+from resources.lib.kodi.items import Items
+from resources.lib.kodi.search_history import SearchHistory
+from resources.lib.kodi.settings import Settings
+from resources.lib.kodi.vfs import VFS
+from resources.routes import *
+import os
+import sys
+import urllib.parse
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+
+addon = xbmcaddon.Addon()
+addon_id = addon.getAddonInfo("id")
+addon_base = "plugin://" + addon_id
+addon_profile_path = xbmc.translatePath(addon.getAddonInfo("profile"))
```

Comment:
xbmcvfs.translatePath for Matrix and aboce

Reply:
Thanks, fixed it.